scripts/ctrl-main.js

```diff
@@ -202,6 +202,10 @@ angular.module('angularTubeApp', ['ngRoute', 'cfp.hotkeys'])  .config(function (
 
         $scope.advancedFilters = false;
 
+        // Initilize with empty film database.
+        $scope.filmDB = [];
```

Comment:
when the code loads filmDB isn't defined yet and it tries to do a slice on undefined.